MMA Removal Utility/README.md

```diff
@@ -262,6 +336,56 @@ let virtualMachinesWithMMAExtensions = virtualMachines
 | project VMResourceID, OSType, ExtensionResourceId, ExtensionType;
 virtualMachinesWithMMAExtensions
 ```
+
+For VMSS(s) having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let vmss = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType contains "VirtualMachineScaleSets"
+| summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
+| extend  OSType = tostring(parse_json(Metadata_s).OSType)
+| project VMResourceID = ResourceId, OSType;
+let vmssMMAExtensions = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType =~ "VMSSExtension" and Source_s =~ "AzTS_05_VMExtensionInventoryProcessor"
+| summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
+| extend VMResourceID = tolower(substring(ResourceId,0,indexof(ResourceId, '/', 0, -1, 9 )))
+| extend ExtensionType = tostring(parse_json(Metadata_s).ExtensionType)
+| where ExtensionType =~ "OmsAgentForLinux" or ExtensionType =~ "MicrosoftMonitoringAgent"
+| project ResourceId, VMResourceID, ExtensionType, ExtensionResourceId = ResourceId;
+let vmssWithMMAExtensions = vmss
+| join kind=inner  (vmssMMAExtensions) on VMResourceID
+| project VMResourceID, OSType, ExtensionResourceId, ExtensionType;
+vmssWithMMAExtensions
+```
+
+For Azure Arc Server(s) having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let hybridVirtualMachines = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType contains "HybridCompute"
```

Comment:
same

Reply:
Added =~ in all the places and tested the query in LA